webui/react/src/pages/SignOut.tsx

```diff
@@ -44,14 +46,15 @@ const SignOut: React.FC = () => {
       authStore.reset();
 
       if (info.externalLogoutUri) {
-        routeAll(info.externalLogoutUri);
+        const isAuthenticated = await checkAuth();
+        if (isAuthenticated) routeAll(info.externalLogoutUri);
```

Comment:
see above re: `checkAuth` return values:
```suggestion
        await checkAuth();
        routeAll(info.externalLogoutUri);
```